.github/workflows/js_code_review.yaml

```diff
@@ -0,0 +1,59 @@
+name: "Code Review"
+
+on:
+  workflow_call: {}
+
+jobs:
+  js_code_review:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+
+      # Corepack is an official tool by Node.js that manages package managers versions
+      - name: Setup yarn
+        run: corepack enable
+
+      - name: Setup Node.js
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version-file: ".node-version"
+          cache: "yarn"
+
+      - name: Install dependencies
+        run: yarn install --immutable
+
+      - name: Setup turbo cache
+        uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2
+        with:
+          path: node_modules/.cache/turbo
+          key: ${{ runner.os }}-turbo-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-turbo-
+
+      # The code-review script must be defined in the root workspace
+      # Example: "turbo run typecheck format:check lint:check //#test:coverage"
+      # See https://github.com/pagopa/io-fims/pull/41
+      - name: Run code-review script
+        run: yarn run code-review
+
+      # Codecov provides reports and metrics about test coverage data.
+      # To enable set CODECOV_TOKEN secret at repo level and make sure
+      # that your "code-review" script creates a "coverage/" folder in
+      # the root workspace.
+      - name: Check if CODECOV_TOKEN is set
+        id: codecov
+        shell: bash
+        run: |
+          if [ "${{ secrets.CODECOV_TOKEN }}" != '' ]; then
```

Comment:
You could santize the secret value with:
```yaml
env:
  CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
run: |
  if [ "${{ env.CODECOV_TOKEN }}" != '' ]; then
```

Reply:
I saw it in some stackoverflow threads, but it's safe? I don't know how Github Workflow handles this step-scoped environments.

If it seems safe to you I will refactor it to remove that extra step!